tests/common/templates/pfc_storm_mlnx_sonic.j2

```diff
@@ -1,5 +1,5 @@
 {% if (pfc_asym  is defined) and (pfc_asym == True) %}
-docker exec syncd /bin/bash -c "{% if pfc_storm_defer_time is defined %} sleep {{pfc_storm_defer_time}} &&{% endif %} python /root/{{pfc_gen_file}} -p {{pfc_queue_index}} -n {{pfc_frames_number}} -i {{pfc_fanout_interface}} -l {{pfc_fanout_label_port}}" > /dev/null 2>&1 &
+docker exec syncd /bin/bash -c "{% if pfc_storm_defer_time is defined %} sleep {{pfc_storm_defer_time}} &&{% endif %} python /root/{{pfc_gen_file}} -p {{pfc_queue_index}} -n {{pfc_frames_number}} -i {{pfc_fanout_interface}} -l {{pfc_fanout_label_port}}" > /dev/null 2>&1
```

Comment:
Hi, Bing, Arista also failed recently. Is it helpful on Mellanox device? I tried, also with "chrt -f 90" or "nice -20", seems not helpful on Arista platform, thanks.

Reply:
This script [tests/common/templates/pfc_storm_mlnx_sonic.j2](https://github.com/sonic-net/sonic-mgmt/pull/14335/files/5c2d3e43e48007e295ae989cba8991f70379a2c0#diff-0101d6778003b9d56b41bc3164d80a3058beb54f923c75fb8e12d3f93bde53dc) is only for Mellanox platform. We can't do this change on Arista platform as the `pfc_gen.py` on Arista platform has to be run in background.